MIP/mip-n/README.md

```diff
@@ -0,0 +1,50 @@
+# MIP-55: The Bilbao Model
+- **Description**: Proposes an AB-FFS L2 model that features a Gas Deficit Minter, Bridge Insurance Account, low-cost DA and settlement, high centralized control, and ultimately high operational assumptions.
+- **Authors**: [Liam Monninger](mailto:liam@movementlabs.xyz)
+- **Reviewer**: Andreas Penzkofer
+- **Desiderata**: $\emptyset$
+
+## Abstract
+
+We define the Bilbao Model as an AB-FFS L2 model that features a Gas Deficit Minter, Bridge Insurance Account, low-cost DA, high centralized control, and ultimately high operational assumptions. At the core of this model is the highly-controlled use of trusted signers with high liquidity to replace components proposed in the [Biarritz Model](https://github.com/movementlabsxyz/MIP/pull/54).
```

Comment:
```suggestion
We define the Bilbao Model as an AB-FFS L2 model that features a Gas Deficit Minter, Bridge Insurance Account, low-cost DA, high centralized control, and ultimately high operational assumptions. At the core of this model is the highly-controlled use of trusted signers with high liquidity. 

It precedes the introduction of governance components proposed in the [Biarritz Model](https://github.com/movementlabsxyz/MIP/pull/54). 
```

Reply:
Not sure the Biarritz should be mentioned here. logically the biarritz model improves the bilbao model, even though historically this may be different.